Apsitvarkom.UnitTests/Api/Controllers/PollutedLocationControllerTests.cs

```diff
@@ -49,71 +59,118 @@ public class PollutedLocationControllerTests
     [SetUp]
     public void SetUp()
     {
-        m_repository = new Mock<ILocationDTORepository<PollutedLocationDTO>>();
-        m_controller = new PollutedLocationController(m_repository.Object);
+        var config = new MapperConfiguration(cfg =>
+        {
+            cfg.AddProfile<PollutedLocationProfile>();
+        });
+        config.AssertConfigurationIsValid();
+        m_mapper = config.CreateMapper();
+
+        m_repository = new Mock<IPollutedLocationRepository>();
+        m_controller = new PollutedLocationController(m_repository.Object, m_mapper, new CoordinatesGetRequestValidator());
     }
 
     [Test]
-    public void Constructor_HappyPath_IsSuccess() => Assert.That(new PollutedLocationController(m_repository.Object), Is.Not.Null);
+    public void Constructor_HappyPath_IsSuccess() => Assert.That(new PollutedLocationController(m_repository.Object, m_mapper, new CoordinatesGetRequestValidator()), Is.Not.Null);
 
     [Test]
-    public async Task GetAll_RepositoryReturnsPollutedLocationDTOs_OKActionResultReturned()
+    public async Task GetAll_RepositoryReturnsPollutedLocations_OKActionResultReturned()
     {
-        m_repository.Setup(self => self.GetAllAsync()).ReturnsAsync(PollutedLocationDTOs);
+        m_repository.Setup(self => self.GetAllAsync())
+            .ReturnsAsync(PollutedLocations);
 
         var actionResult = await m_controller.GetAll();
 
+        Assert.That(actionResult.Result, Is.TypeOf <OkObjectResult>());
```

Comment:
```suggestion
        Assert.That(actionResult.Result, Is.TypeOf<OkObjectResult>());
```
you have written it without the space elsewhere:D